### ISSUE TITLE
Install script should remove Alcatraz from skipped bundles list

### ISSUE THREAD

**Issue by Ashton-W** [alcatraz/Alcatraz#281]
Since **Xcode 6.3**(?) you are given a single chance to allow Xcode to load the Alcatraz bundle, if you choose not to the preference is saved in defaults.

```
defaults read com.apple.dt.Xcode.plist DVTPlugInManagerNonApplePlugIns-Xcode-6.3.2
```
This command shows structure of this preference.
```
{
    allowed =     {
        "com.mneorr.Alcatraz" =         {
            version = 1;
        };
        "ru.DeepIT.XcodeColors" =         {
            version = 109;
        };
    };
    skipped =     {
    };
}
```

Should the Alcatraz install script check or fix this preference to allow Alcatraz to be loaded again?

**Comment by pbendersky** [comment]
:+1: I was just fighting this, and kept reinstalling Alcatraz without success.

**Comment by supermarin** [comment]
Thanks for the suggestion, this one sounds good :+1:

**Comment by pbendersky** [comment]
Just created a PR for this (#282)

**Comment by guillaume-algis** [comment]
Fixed in #315
